circle.yml

```diff
@@ -0,0 +1,18 @@
+dependencies:
+  pre:
+    - |
+        sudo apt-get install libsdl2-dev libfreetype6-dev
+
+        if [ ! -d ~/bam ]; then
+          git clone https://github.com/matricks/bam.git ~/bam
+          git reset --hard f012dd9a3e38295b8a45af5a101d29573381f169
```

Comment:
Specifically checks out the [currently latest commit](https://github.com/matricks/bam/commit/f012dd9a3e38295b8a45af5a101d29573381f169) for bam that does not have a tag yet (instead of 0.5.0).